client/src/modules/fiscal/fiscal.openingBalance.html

```diff
@@ -37,15 +37,18 @@
           <div ng-class="{'text-success': FiscalOBCtrl.balanced, 'text-danger': !FiscalOBCtrl.balanced}">
             <span ng-show="OpeningAccountForm.$submitted" >
               <i class="fa" ng-class="{'fa-check': FiscalOBCtrl.balanced, 'fa-warning': !FiscalOBCtrl.balanced}"></i>
-              <span data-status="balanced" ng-show="FiscalOBCtrl.balanced">{{ 'ACCOUNT.BALANCED' | translate }}</span>
-              <span data-status="not-balanced" ng-show="!FiscalOBCtrl.balanced && FiscalOBCtrl.hasPositive">{{ 'ACCOUNT.NOT_BALANCED' | translate }}</span>
-              <span data-status="not-positive" ng-show="!FiscalOBCtrl.balanced && !FiscalOBCtrl.hasPositive">{{ 'ACCOUNT.NOT_POSITIVE' | translate }}</span>
+              <span data-status="balanced" ng-show="FiscalOBCtrl.balanced" translate>ACCOUNT.BALANCED</span>
+              <span data-status="not-balanced" ng-show="!FiscalOBCtrl.balanced">
+                <span translate>ACCOUNT.NOT_BALANCED</span>
+                (<span translate>FORM.LABELS.DEBIT</span>: {{ FiscalOBCtrl.totalDebit | currency: FiscalOBCtrl.enterprise.currency_id }} &nbsp;
+                <span translate>FORM.LABELS.CREDIT</span>: {{ FiscalOBCtrl.totalCredit | currency: FiscalOBCtrl.enterprise.currency_id }})
+              </span>
             </span>
           </div>
 
           <button type="submit" data-method="submit" ng-disabled="!FiscalOBCtrl.editBalanceEnabled && !FiscalOBCtrl.previousFiscalYearExist" class="btn btn-primary">
-            <span ng-show="!FiscalOBCtrl.previousFiscalYearExist">{{ 'FORM.BUTTONS.SUBMIT' | translate }}</span>
-            <span ng-show="FiscalOBCtrl.previousFiscalYearExist">{{ 'FORM.BUTTONS.IMPORT' | translate }}</span>
+            <span ng-show="!FiscalOBCtrl.previousFiscalYearExist" translate>FORM.BUTTONS.SUBMIT</span>
+            <span ng-show="FiscalOBCtrl.previousFiscalYearExist" translate>FORM.BUTTONS.IMPORT</span>
```

Comment:
This is a good update :+1:.  

It would make sense to have the text say "Import Previous Closing Balance" or "Import Previous Balances" or something.  At the moment, "Import" is very terse (not well explained).